src/FakeItEasy/Core/FakeWrapperConfigurator.cs

```diff
@@ -38,9 +38,9 @@ public override IFakeOptionsForWrappers<T> Wrapping(T wrappedInstance)
             return this.fakeOptions.Wrapping(wrappedInstance);
         }
 
-        public override IFakeOptions<T> WithAdditionalAttributes(IEnumerable<CustomAttributeBuilder> customAttributeBuilders)
+        public override IFakeOptions<T> WithAdditionalAttributes(params Expression<Func<Attribute>>[] customAttributeExpressions)

```

Comment:
I don't think "factories" would be misleading, it would accurately convey the content of the expressions. However, the point is purely academic if we're not going to use that convention, and I still think it's not necessary to do so.